### PR TITLE
Add typings for link2aws

### PR DESCRIPTION
**Pull request by kpeters-cbsi**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "@definitelytyped/dtslint/dt.json" }`, and no additional rules.
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/link2aws/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+// Definitions by: Kit Peters <https://github.com/popefelix>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export class ARN {
+  constructor(text: string);
+  string: string;
+  console: string;
+  qualifiers: string[];
+  pathLast: string;
+  consoleLink: string;
```
Comment (MysteryBlokHed) [inline types/link2aws/index.d.ts:12]: These properties should probably use the `get` syntax to match the source code:

```suggestion
  get string(): string;
  get console(): string;
  get qualifiers(): string[];
  get pathLast(): string;
  get consoleLink(): string;
```

File: types/link2aws/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
+// Type definitions for link2aws 1.0
+// Project: https://github.com/link2aws/link2aws
+// Definitions by: Kit Peters <https://github.com/popefelix>
```
Comment (MysteryBlokHed) [inline types/link2aws/index.d.ts:3]: This doesn't match your GitHub username

**Review by MysteryBlokHed — CHANGES_REQUESTED** [review]

**Comment by kpeters-cbsi** [comment]
I'll make those changes and resubmit with my other GH username.

**Comment by MysteryBlokHed** [comment]
The PR could still be merged with this account as the author. Just need 
a comment from the @popefelix account confirming that it's all right 
being the definition owner 😊

**Comment by kpeters-cbsi** [comment]
Well, that's easy enough 😁

**Comment by PopeFelix** [comment]
I'm popefelix and I approve of this PR. I'm a big fan!

**Comment by PopeFelix** [comment]
But I still can't update the code right now. 😁

File: types/link2aws/index.d.ts (hunk 2 above)
Comment (PopeFelix) [reply types/link2aws/index.d.ts:3]: It's ok by me. I and @kpeters-cbsi are one and the same.

**Comment by MysteryBlokHed** [comment]
No worries. I'll get notified whenever new commits are pushed, and I'll 
take another look then 😄

**Comment by PopeFelix** [comment]
As it turns out I can do all of this from my phone. Ain't technology grand? 😁

**Review by MysteryBlokHed — APPROVED** [review]
LGTM! Just needs a maintainer to approve as well and it'll get merged

**Comment by PopeFelix** [comment]
The bot thinks I'm a maintainer. Lol

**Review by andrewbranch — APPROVED** [review]

**Comment by PopeFelix** [comment]
Ready to merge

**Comment by kpeters-cbsi** [comment]
Ready to merge

**Pull request merged**